src/documentation/pages/components/date-picker/examples/_calendar-active.vue

```diff
@@ -0,0 +1,75 @@
+<template lang="pug">
+  <ComponentExample titleSize="h4" title="Active" id="calendar-active" :tabs="exampleTabs">
+    chi-date(value="05/15/2019" slot="example")
+    <pre class="language-html" slot="code-webcomponent">
+      <code v-highlight="$data.codeSnippets.webcomponent" class="html"></code>
+    </pre>
+    <Wrapper slot="code-htmlblueprint">
+      <JSNeeded />
+        .chi-tab__description Use class <code>-active</code> for the selected date
+      <pre class="language-html">
+        <code v-highlight="$data.codeSnippets.htmlblueprint" class="html"></code>
+      </pre>
+    </Wrapper>
+  </ComponentExample>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+@Component({
+  data: () => {
+    return {
+      datePickerDates: Array.from(Array(30).keys()),
+      datePickerDayNames: ['S', 'M', 'T', 'W', 'T', 'F', 'S'],
+      exampleTabs: [
+        {
+          active: true,
+          id: 'webcomponent',
+          label: 'Web component',
+        },
+        {
+          id: 'htmlblueprint',
+          label: 'HTML blueprint',
+        },
+      ],
+      codeSnippets: {
+        webcomponent: `<chi-date value="05/15/2019"></chi-date>`,
+        htmlblueprint: ``,
+      },
+    };
+  },
+})
+export default class CalendarActive extends Vue {
```

Comment:
The HTML Blueprint code snippet is missing `<div class="chi-datepicker__day ">31</div>`

![image](https://user-images.githubusercontent.com/86958774/165541829-72056d39-fd1d-43d7-ba89-6060a9a664e1.png)

Reply:
Good catch!